pywbem_mock/_mainprovider.py

```diff
@@ -811,152 +790,247 @@ def remove_namespace(self, namespace):
     #
     ####################################################################
 
-    def EnumerateClasses(self, **params):
-        # pylint: disable=invalid-name
+    def EnumerateClasses(self, namespace, ClassName=None,
+                         DeepInheritance=None, LocalOnly=None,
+                         IncludeQualifiers=None, IncludeClassOrigin=None):
         """
-        Implements a mock server responder for
+        Implements a CIM server responder for
         :meth:`~pywbem.WBEMConnection.EnumerateClasses`.
 
-        Enumerate classes from class repository. If classname parameter
+        Enumerate classes from datastore. If classname parameter
```

Comment:
Suggest to change "datastore" to "CIM repository".

Occurs multiple times.

Reply:
Changed everywhere.